ui/src/common/components/ListWithPagination.jsx

```diff
@@ -41,14 +50,18 @@ class ListWithPagination extends Component {
         pageSize={pageSize}
         loading={loading}
         showTotal={ListWithPagination.getPaginationRangeInfo}
+        onShowSizeChange={onSizeChange}
```

Comment:
below props should only be set if this is one (`onShowSizeChange`) is present, or at least do `showSizeChanger={onShowSizeChange != null}`, otherwise size changer will be displayed no matter what.